Sources/Bundle/BundledModule.swift

```diff
@@ -0,0 +1,33 @@
+/*********************************************
+ *
+ * This code is under the MIT License (MIT)
+ *
+ * Copyright (c) 2021 AliSoftware
+ *
+ *********************************************/
+
+import Foundation
+
+/// Declaring your `Reusable` based classes to conform to `BundledModule`.
+/// This provides a default implementation that uses the bundle associated
+///  with a Swift Package Manager Bundle (`Bundle.module`).
+///
+/// Example:
+/// ```
+/// final class CustomCell: UITableViewCell, Reusable, BundledModule { ... And that's it! ... }
+/// ```
+///
+/// For the the resource to be loaded from the bundle of your package, you
+/// will also need to add the following extension somewhere in your package
+/// framework...
+///
+/// ```
+/// public extension BundledModule {
+///    static var bundle: Bundle {
+///        Bundle.module
+///    }
+/// }
+/// ```
+
+public  protocol BundledModule: AnyObject, Bundled {
```

Comment:
I wonder if it's really worth making `Reusable` itself provide this protocol, given that we don't provide a default implementation for it (I think that's because we can't do it in a way that would make it still work and compile if not integrated via SPM? Though I'll have to re-read the SE proposal about SPM Resource Bundles again as I thought they made a point trying to make this work…)? 🤔 

If we can't provide a default implementation for those ☝️  reasons, and we have to let the users of `Reusable` declare the default implementation themselves, I'd say it's probably simpler to let them declare the protocol too (in addition to the default implementation), that way:

1. They won't be surprised if it does work if they conform their class to our `BundleModule` but forgot to follow the instructions to provide the default implementation
2. They could choose their own name (in case e.g. that name conflicts with a protocol or class of the same name in their own codebase or if they prefer something else)